packages/react-aria-components/src/Collection.tsx

```diff
@@ -668,7 +668,7 @@ export function useCachedChildren<T extends object>(props: CachedChildrenOptions
         res.push(rendered);
       }
       return res;
-    } else {
+    } else if (typeof children !== 'function') {
```

Comment:
This code is making it more explicate that you can't have `children` as a function when `items` is not defined, right? Just double checking that this isn't messing with the case where someone has defined `items` and `children` is not a function. I know this second case is "wrong", but I saw it in a support question. Trying to avoid a breaking change.

Reply:
Yes. Children is only allowed as a function when `items` is defined. Otherwise it won't work because it'll pass the function through as the child of a div. And nothing here is breaking because this is only affecting RAC for now.